examples/pipelines/windows10-installer/README.md

```diff
@@ -0,0 +1,122 @@
+# Windows 10 Installer Pipeline
+
+Downloads a Windows Source ISO into a DV and installs Windows into a new base DV.
+It will then spin up an installation VM and use Windows Answer Files to automatically install the VM.
+Then the pipeline will wait for the installation to complete and will delete the installation VM while keeping the artifact DV with the installed operating system.
+The pipeline can be customized to support different installation requirements.
+
+There is a specific version of this pipeline for OKD. This version is using templates, which are not available on Kubernetes.
+
+## Prerequisites
+
+- KubeVirt `v0.53.1`
+- Tekton Pipelines `v0.35.0`
+
+## Links
+
+- [Windows 10 Installer Pipeline for Kubernetes](https://github.com/kubevirt/tekton-tasks-operator/blob/main/data/tekton-pipelines/kubernetes/windows10-installer.yaml)
+- [Windows 10 Installer Pipeline for OKD](https://github.com/kubevirt/tekton-tasks-operator/blob/main/data/tekton-pipelines/okd/windows10-installer.yaml)
+- For example PipelineRuns see commit message of [2c4daed](https://github.com/kubevirt/tekton-tasks-operator/commit/2c4daed4124654a765f69acdc2b1c7390ee3c2f4)
+
+### Obtain Windows ISO download URL
+
+1. Go to https://www.microsoft.com/en-us/software-download/windows10ISO.
+   You can also obtain a server edition for evaluation at https://www.microsoft.com/en-us/evalcenter/evaluate-windows-server-2019. (needs a different answers file!)
+2. Fill in the edition and `English` language (other languages need to be updated in windows10-autounattend ConfigMap) and go to the download page.
+3. Right-click on the 64-bit download button and copy the download link. The link should be valid for 24 hours.
+4. Initialize a WIN_URL variable that will be used to create a DataVolume which will download this ISO into a PVC.
+
+```bash
+# Real URL can look differently
+WIN_URL="https://software.download.prss.microsoft.com/db/Win10_21H2_English_x64.iso..."
+```
+
+### Prepare autounattend.xml ConfigMap
+
+1. Supply, generate or use the default autounattend.xml.
```

Comment:
Would be great if this information could be in https://kubevirt.io/user-guide/virtual_machines/startup_scripts/#sysprep, which could be referenced here.

Reply:
I changed the docs and created https://github.com/kubevirt/user-guide/pull/542